src/index.html

```diff
@@ -6,17 +6,431 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <title>Nothing</title>
+
+    <!-- #region links -->
+    <link
+      rel="icon"
+      href="images/logo-icon.svg"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300..700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:opsz,wght@14..32,700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="styles/main.scss"
     />
+    <!-- #endregion -->
+
     <script
       src="scripts/main.js"
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <!-- #region header -->
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a
+              href="#"
+              class="top-bar__logo-link"
+            >
+              <img
+                class="top-bar__logo"
+                src="./images/logo-2x.png"
+                alt="Nothing logo"
+              />
+            </a>
+
+            <div class="top-bar__icons">
+              <a
+                href="tel:+1 234 555-55-55"
+                class="icon icon--phone"
+              ></a>
+
+              <a
+                href="#menu"
+                class="icon icon--menu"
+              ></a>
+            </div>
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Bring joy back
+              <br />
+              to the everyday
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+    <!-- #endregion -->
+
+    <!-- #region aside -->
+    <aside
+      class="page__menu menu"
+      id="menu"
+    >
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a
+            href="#"
+            class="top-bar__logo-link"
+          >
+            <img
+              class="top-bar__logo"
+              src="images/logo-2x.png"
+              alt="Nothing logo"
+            />
+          </a>
+          <a
+            href="#"
+            class="icon icon--close"
+          ></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
```

Comment:
When clicking on each Item that navigates by ID, I want to see a smooth scroll, not a sharp flicker. Change it